### PR TITLE
Improve staking implementation docs

### PR DESCRIPTION
**Pull request by leoneparise**

### CODE REVIEW

File: platform/staking.md [hunk 1]
```diff
@@ -5,35 +5,110 @@ a set of open source libraries and documentation to help blockchain developers t
...
-    - Validators list
-    - Staking pool (if applicable)
-    - Current annual APR
-  - After finishing the implementation, open a pull request to [blockatlas](https://github.com/trustwallet/blockatlas) project.
-  You can find an example for Cosmos [here](https://github.com/trustwallet/blockatlas/tree/master/platform/cosmos)
-
-## Step 4: Define Validators Information
-  - Open a pull request to [trust assets](https://github.com/trustwallet/assets) and add all validators
+## Step 1: Talk about your implementation
+Create an issue in [Trust Platform](https://github.com/trustwallet/web-core/tree/master/packages/rpc/src/cosmos) describing your poject. Please fill the following questions before proceding with your implementation:
```
Comment (hewigovens) [inline platform/staking.md]:
```suggestion
Create an issue in [Trust Platform](https://github.com/trustwallet/web-core/issues) describing your poject. Please fill the following questions before proceding with your implementation:
```

**Comment by hewigovens** [comment]
it would be nice to have a simple check list

**Comment by leoneparise** [comment]
> it would be nice to have a simple check list

Good idea. I'm on it now

File: platform/staking.md [hunk 2]
```diff
@@ -5,35 +5,114 @@ a set of open source libraries and documentation to help blockchain developers t
...
+type DelegationStatus string
+
+const (
+  DelegationStatusActive  DelegationStatus = "active"
+  DelegationStatusPending DelegationStatus = "pending"
+)
+
+type StakeValidatorInfo struct {
+  Name        string `json:"name"`
+  Description string `json:"description"`
```
Comment (vikmeup) [inline platform/staking.md]: can you point to go lang code instead? it might change a bit and it will get oudated

Comment (leoneparise) [reply platform/staking.md]: Ok. I'll remove the models sample and just tell them to check it out

**Pull request merged**